src/components/color_picker/color_picker_entry.xml

```diff
@@ -0,0 +1,20 @@
+<templates>
+  <t t-name="o-spreadsheet-ColorPickerEntry" owl="1">
+    <div class="o-dropdown o-color-picker-entry">
+      <div
+        class="o-tool o-with-color"
+        t-on-click.stop="props.onClick"
+        t-att-title="props.tooltipText">
+        <span t-att-style="iconStyle">
+          <t t-call="{{props.icon}}"/>
+        </span>
+      </div>
+      <ColorPicker
+        t-if="props.showColorPicker"
```

Comment:
(didn't check the code too much so it may not be possible)

Can we put all the logic to show/hide the ColorPicker inside the `ColorPickerEntry` ?